src/html-requires-loader.ts

```diff
@@ -9,6 +9,8 @@ function loader(this: Webpack.Loader, content: string) {
       const attrs = loader.attributes[tag];
       return attrs && attrs.includes(attr);
     })
+    // Ignore values that contain interpolated values
+    .filter(attr => !/(^|[^\\])\$\{/.test(attr.value))
```

Comment:
I guess we could. Note that all variations of `view.bind` & co. are also ignored.

I am unsure if the dependencies should be handled at the usage site (e.g. a dynamic compose) or in the class that exports the members used to build the view.

Ideally, I think you want your class to look like `class X { template: moduleName("components/cats") }`. I think that's clean and creates dependencies on good modules.

Also one thing I don't quite like with a globbing like approach to dynamic dependencies (similar to what webpack actually does), is that it tends to grab more than needed.